.readthedocs.yaml

```diff
@@ -0,0 +1,24 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required 
+version: 2
+
+# Build all formats (htmlzip, pdf, epub)
+#formats: all
+formats: []
+
+# Optionally set the version of Python and requirements required to build your
+# docs
+python:
+   version: 3.7
```

Comment:
Is Python 3.7 required for this? The requirement for running the actual wrappers is 3.6.3.